src/snowcli/cli/streamlit/commands.py

```diff
@@ -63,34 +62,33 @@ def streamlit_describe(
 @with_output
 @global_options_with_connection
 def streamlit_create(
-    name: str = typer.Argument(..., help="Name of streamlit to create."),
-    file: Path = typer.Option(
-        "streamlit_app.py",
-        exists=True,
-        readable=True,
-        file_okay=True,
-        help="Path to the Streamlit Python application (`streamlit_app.py`) file.",
-    ),
-    from_stage: Optional[str] = typer.Option(
-        None,
-        help="Stage name from which to copy a Streamlit file.",
-    ),
-    use_packaging_workaround: bool = typer.Option(
-        False,
-        help="Whether to package all code and dependencies into a zip file. Valid values: `true`, `false` (default). You should use this only for a temporary workaround until native support is available.",
+    streamlit_names: List[str] = typer.Argument(
+        ..., help="Streamlits names. Multiple can be provided."
     ),
     **options,
 ) -> CommandResult:
     """
     Creates a new Streamlit application object in Snowflake. The streamlit is created in database and schema configured in the connection.
     """
-    cursor = StreamlitManager().create(
-        streamlit_name=name,
-        file=file,
-        from_stage=from_stage,
-        use_packaging_workaround=use_packaging_workaround,
-    )
-    return SingleQueryResult(cursor)
+    dm = DefinitionManager()
+    streamlits = dm.project_definition.get("streamlits")
```

Comment:
I think we should have one streamlit per project

Reply:
Changed